### ISSUE TITLE
Hip-hop Webpack + Angular 6.0.1

### ISSUE THREAD

**Issue by isopen** [webpack/webpack#7292]
`npm run-script build`

Guys in general, I'm already too lazy to understand. But I tried to understand and understood that the old config is not good at all.
Config:
https://github.com/isopen/soc/blob/master/frontend/webpack.config.js
```
WARNING in ./node_modules/@angular/core/fesm5/core.js
System.import() is deprecated and will be removed soon. Use import() instead.
For more info visit https://webpack.js.org/guides/code-splitting/
 @ ./src/main.ts 5560:15-36 1:0-47 6:4-18
 @ multi ./src/main.ts
WARNING in ./node_modules/@angular/core/fesm5/core.js
System.import() is deprecated and will be removed soon. Use import() instead.
For more info visit https://webpack.js.org/guides/code-splitting/
 @ ./src/main.ts 5572:15-102 1:0-47 6:4-18
 @ multi ./src/main.ts

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  main.bundle.js (1.23 MiB)
  assets/css/awesome/fonts/fontawesome-webfont.svg (434 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (1.23 MiB)
      main.bundle.js

WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
```

Everything happened after switching to Angular 6.0.1. But Angular himself has nothing to do with it.
```
ng serve
ng build --aot --optimization
```
Works great.
But I also want to work fine and through the webpack. Show me where I need to fix the configuration or packages to work as before. Thanks :)

**Comment by isopen** [comment]
Another such moment. If you insert such a piece of code, then the webpack swears.
```
async function test123() {
  for await (const i of [1, 2, 3]) {
    console.log(123);
  }
}
test123() here a return from promise.
```
`ng serve` behaves adequately with the current implementation :)

**Comment by montogeek** [comment]
It looks like you just deleted our lovely crafted [issue template](https://raw.githubusercontent.com/webpack/webpack/master/.github/ISSUE_TEMPLATE.md). It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please either update the issue with the template and reopen, or open a new issue.

**Comment by isopen** [comment]
This is by template other.Open please the issue

**Comment by montogeek** [comment]
What is your issue?

**Comment by isopen** [comment]
I need to fix the config so that there are no more warnings and es2018 was going to.
Yesterday I did everything I could. Now it's your turn.

It is the webpack. ng works. thanks:*)

**Comment by ooflorent** [comment]
I don’t see the problem here. These are just warnings. If you need support, please ask your question on dedicated channels such as stackoverflow or our gitter chat.

**Comment by ooflorent** [comment]
Regarding the async iterator, please open a new issue. 

If you want to disable the size warnings, add `performance: false` to your configuration.

**Comment by isopen** [comment]
I agree that the last 2 varings seem to be okay.
But the first inconsistency is not very good :( While it is not clear how to close it correctly.

**Comment by isopen** [comment]
I'm completely satisfied with the `ng` build system. For webpack I'm not in a hurry to write a config.
I expect when stability begins in the settings of the webpack. Then I'll create a configuration for Angular.
The aim of the project is not to save on bundle. The purpose of the project is testing of possible version migrations. It was quite simple and intuitively understandable to migrate to a Angular 6.
I have already found an article https://blog.angular.io/version-6-of-angular-now-available-cc56b0efa7a4
for migrating. But it turned out to be no longer relevant. On an intuitive level, I already migrated to a newer version. At the moment I do not want to collect another bicycle with webpack.
I like webpack, but not so much to change every month build configs.
